pages/common/xmllint.md

```diff
@@ -1,24 +1,24 @@
 # xmllint
 
-> XML parser and linter.
+> XML parser and linter that supports XPath, a syntax for navigating XML trees.
 
-- Return all nodes named "foo":
+- Return all nodes (tags) named "foo":
 
 `xmllint --xpath "//{{foo}}" {{source_file.xml}}`
 
-- Return as string the contents of first node named "foo":
+- Return, as a string, the contents of the first node named "foo":
```

Comment:
I thought about that first, but wondered if it made sense to leave the "string" so late in the description since it's the first part of the xpath expression.

I don't think it's a big deal, though. Let me know if you don't think this concern outweighs the admittedly more awkward sentence construction I chose, in which case I'll make the change.